### ISSUE TITLE
Support newer time

### ISSUE THREAD

**Issue by kozross** [haskellfoundation/shields#1]
Currently blocked by [`io-streams`](https://github.com/snapframework/io-streams/issues/78) and [`snap-core`](https://github.com/snapframework/snap-core/issues/306).

**Comment by kozross** [comment]
No longer relevant as of dd9d1841844e0472af00da94efbf21a3e7ef2873, as we no longer depend on it.
